src/components/Common/FilePreviewDialog.tsx

```diff
@@ -173,6 +236,17 @@ const FilePreviewDialog = (props: FilePreviewProps) => {
                 </div>
               )}
             </div>
+
+            {uploadedFiles &&
+              uploadedFiles.length > 1 &&
+              index < uploadedFiles.length - 1 && (
+                <ButtonV2
```

Comment:
<img width="232" alt="image" src="https://github.com/user-attachments/assets/7e05bed6-a27a-4963-ba4d-63aa7b90a642">

Adjust the margin b/w this button and the image.

Reply:
done